OperatingSystem-Services/Platform-Linux/1-Ansible/First-15/Backups/ansible.cfg

```diff
@@ -0,0 +1,6 @@
+[defaults]
+remote_user=blueteam
+inventory = infra.yml
+host_key_checking = False
+private_key_file = ~/.ssh/Cyber-Key
```

Comment:
We should make a note for everyone to copy the same key or know change this private key file, during our first 15 minutes, when using this script and others like it.

Reply:
If everyone generates a ssh key with the same name (or atleast the file is the same) assuming we add everyone's key to the authorized_key file for the user we will execute ansible playbooks through then it should be fine. As you said this is something we would want to make note of. Either in the documentation or first 15 min docs.